scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -56,7 +56,7 @@ setTheme() {
 # Do NOT modify code below           #
 ######################################
 
-GLV_VERSION=v1.23.1
+GLV_VERSION=v1.23.2
```

Comment:
Looks great, only thing as this is a bigger change and not a fix/patch, would be to bump version to 1.24.0 instead.

Reply:
Sounds good, looking forward to the release.